internal/pkg/service/common/etcdop/op/atomic_test.go

```diff
@@ -251,80 +251,82 @@ func TestAtomicOp(t *testing.T) {
 	}
 
 	for _, tc := range cases {
-		tc.Run(t)
+		tc := tc
```

Comment:
Here is a bug :see_no_evil: - missing `tc := tc`.

I added `t.Parallel()` at the work end, so I didn't notice it.

I realized it during next work. The last test-case was started repeatedly, instead of other test cases.

Linter didn't reveal it, because callbacks are wrapped in methods.

The solution is to add `tc := tc` or remove the pointer receiver from `*atomicOpTestCase` methods. 

I did both, and tweaked it a bit.

![image](https://github.com/keboola/keboola-as-code/assets/19371734/78038d2f-66fb-43d3-8537-8809264c659f)